react/components/MiniCartContent.js

```diff
@@ -206,10 +206,10 @@ class MiniCartContent extends Component {
     const discount = this.calculateDiscount(items, orderForm.value)
     return (
       <Fragment>
-        <div className={classes}>
+        <main className={classes}>
```

Comment:
shouldn't the `main` tag only be used to describe the main content of the **document**?

Reply:
I don't think this can be a `main` element, since you should only put one `main` element in the whole page [according with mdn](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/main)

Reply:
What context i should use, the component or the document (all page)? this is a real doubt that i have: all the literature that i look only use the concept of document to explain the meaning of the tags. but in our case we use components which, for me, can be seen like document.